client/packages/common/src/ui/layout/tables/utils/ColumnDefinitionSetBuilder.ts

```diff
@@ -180,6 +180,7 @@ const getColumnLookup = <T extends RecordWithId>(): Record<
     label: 'label.code',
     key: 'code',
     width: 20,
+    Cell: TooltipTextCell,
```

Comment:
Didn't you previously had it for 'itemCode' as well?

Reply:
Oh don't know why it didn't go through 👀 ... because it was in my stash 🤦🏻